tests/unit_tests/expected_md/details_noregressions.md

```diff
@@ -0,0 +1,3 @@
+Conbench has details about 2 total run(s) on this commit.
+
+This report was generated using a z-score threshold of 500. A regression is defined as a benchmark exhibiting a z-score higher than the threshold in the "bad" direction (e.g. down for iterations per second; up for total time taken).
```

Comment:
Not in this PR, but I made https://github.com/conbench/conbench/issues/399 for documenting+pointing to that documentation for things like this